.circleci/config.yml

```diff
@@ -106,3 +108,17 @@ jobs:
             docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_BRANCH
             docker push quay.io/plotly/image-exporter:$CIRCLE_SHA1
             docker push quay.io/plotly/image-exporter:$CIRCLE_BRANCH
+
+  electron-pack-and-release:
+    docker:
+      - image: circleci/node:8-browsers
+    steps:
```

Comment:
I'd add the same steps to save and restore caches for `.git` and `node_modules` as those in the job `test-node-v8`.

Reply:
@n-riesco good call. Okay, I have added these caches: https://circleci.com/gh/plotly/image-exporter/1201#config/containers/0. I ran the builds twice after this change, to ensure it worked with and without a primed cache—all good. What do you think

Reply:
You are correct, @n-riesco . I have changed the CircleCI caching strategy as per your recommendations, and tested with fresh and primed caches.